redash/handlers/base.py

```diff
@@ -66,10 +66,11 @@ def require_fields(req, fields):
 
 
 def get_object_or_404(fn, *args, **kwargs):
-    try:
-        return fn(*args, **kwargs)
-    except DoesNotExist:
+    rv = fn(*args, **kwargs)
+    if rv is None:
```

Comment:
Maybe we can get rid of this helper, as it seems that Flask-SQLAlchemy has its own: [first_or_404](http://flask-sqlalchemy.pocoo.org/2.1/api/#flask.ext.sqlalchemy.BaseQuery.first_or_404) and [get_or_404](http://flask-sqlalchemy.pocoo.org/2.1/api/#flask.ext.sqlalchemy.BaseQuery.get_or_404).